homeassistant/components/huisbaasje/const.py

```diff
@@ -49,31 +50,62 @@
         "name": "Huisbaasje Current Power",
         "device_class": DEVICE_CLASS_POWER,
         "source_type": SOURCE_TYPE_ELECTRICITY,
-        "state_class": STATE_CLASS_MEASUREMENT,
     },
     {
-        "name": "Huisbaasje Current Power In",
+        "name": "Huisbaasje Current Power In Peak",
```

Comment:
For a future PR: This list that describes sensors, can be replaced by a tuple of `SensorEntityDescription` classes. That will make this much cleaner to work with and allows for the removal of quite a bit of boilerplate code.